config/environments/development.rb

```diff
@@ -26,8 +26,21 @@
     config.cache_store = :null_store
   end
 
+
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "gmail.com",
+    authentication: "plain",
+    enable_starttls_auto: true,
+    user_name: 'veterans360mailer@gmail.com',
```

Comment:
Could you keep quote types consistent. Also @lbkchen which email should we use here?